src/appshell/view/internal/splashscreen.cpp

```diff
@@ -43,16 +45,34 @@ static const QColor versionNumberColor("#22A0F4");
 static constexpr qreal versionNumberSpacing = 5.0;
 
 SplashScreen::SplashScreen()
-    : QSplashScreen(QPixmap(splashScreenSize)),
+    : QWidget(nullptr, Qt::SplashScreen | Qt::FramelessWindowHint),
     m_backgroundRenderer(new QSvgRenderer(imagePath, this))
 {
     setAttribute(Qt::WA_TranslucentBackground);
+    setSize(splashScreenSize);
 
     // Can't make translatable, because translation system not yet initialized
-    showMessage("Loading…");
+    m_message = "Loading…";
```

Comment:
Still true that translations are not yet intialized? ISTR that this is now done quite a bit earlier?

Reply:
#12712 has not been merged yet...

Reply:
Would it make a difference? Is so I'd make the text translatable here already.